pom.xml

```diff
@@ -30,15 +30,17 @@
     </developers>
 
     <properties>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.source>1.8</maven.compiler.source>
+        <java.release>8</java.release>
```

Comment:
I assume that if you specify a property on the command line it automatically overrules the one that's included in the file?

Reply:
That's right